src/components/tabs/tabs.stories.ts

```diff
@@ -70,7 +70,7 @@ simpleDarkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault };
 
 export const bordered = (): string => html`
   <calcite-tabs
-    layout="inline"
+    layout="${select("layout", ["inline", "center"], "inline")}"
```

Comment:
Should we add a visual screenshot for when the layout is "center" to capture the bordered state?